src/components/stepper.tsx

```diff
@@ -0,0 +1,86 @@
+import { Button } from "@mui/material";
+import Box from "@mui/material/Box";
+import Step from "@mui/material/Step";
+import StepButton from "@mui/material/StepButton";
+import StepLabel from "@mui/material/StepLabel";
+import Stepper from "@mui/material/Stepper";
+
+export enum StepStatus {
+  Incomplete = "incomplete",
+  Completed = "completed",
+  Error = "error",
+}
+
+export interface StepperProps {
+  stepTitles: string[];
+  stepStatuses: StepStatus[];
```

Comment:
typo stepStatus

Reply:
I named it like that because it's an array.